learn/advanced/asynchronous_operations.md

```diff
@@ -32,75 +32,93 @@ All of Meilisearch's asynchronous operations belong to a category called "tasks"
-|------------|---------|---------------------------------         |
-| `taskUid`        | integer | Unique sequential identifier             |
-| `indexUid`   | string  | Unique index identifier (always `null` for dumps)                  |
-| `status`     | string  | Status of the task. Value is `enqueued`  |
-| `type`       | string  | Type of task                             |
-| `enqueuedAt` | string  | Represents the date and time in the RFC 3339 format when the task has been `enqueued`                                                        |
+| Field            | Type    | Description                                                                           |
+| :--------------- | :------ | :------------------------------------------------------------------------------------ |
+| **`taskUid`**    | Integer | Unique sequential identifier                                                          |
+| **`indexUid`**   | String  | Unique index identifier (always `null` for [global tasks](#global-tasks))             |
+| **`status`**     | String  | Status of the task. Value is `enqueued`                                               |
+| **`type`**       | String  | Type of task                                                                          |
+| **`enqueuedAt`** | String  | Represents the date and time in the RFC 3339 format when the task has been `enqueued` |
 
 You can use this `taskUid` to get more details on [the status of the task](/reference/api/tasks.md#get-one-task).
 
+### Global tasks
+
+Some task types are not associated with a particular index but apply to the entire instance. These tasks are called global tasks. Global tasks display `null` for the `indexUid` field.
+
+Meilisearch considers the following task types as global:
+
+- [`dumpCreation`](/reference/api/tasks.md#dumpcreation)
+- [`taskCancelation`](/reference/api/tasks.md#taskcancelation)
+- [`taskDeletion`](/reference/api/tasks.md#taskdeletion)
+- [`indexSwap`](/reference/api/tasks.md#indexswap)
+
 ### Task `status`
 
-Task responses always contain a field indicating the request's current `status`. This field has four possible values:
+Task responses always contain a field indicating the request's current `status`. This field has one of the following possible values:
 
-- `enqueued`: the task request has been received and will be processed soon
-- `processing`: the task is being processed
-- `succeeded`: the task has been successfully processed
-- `failed`: a failure occurred when processing the task. No changes were made to the database
+- **`enqueued`**: the task request has been received and will be processed soon
+- **`processing`**: the task is being processed
+- **`succeeded`**: the task has been successfully processed
+- **`failed`**: a failure occurred when processing the task. No changes were made to the database
+- **`canceled`**: the task was canceled
+- **`deleted`**: the task was deleted
```

Comment:
There is no `deleted` status. A deleted task can't be retrieved on the `/tasks` endpoint.